### ISSUE TITLE
For models with batch_norm, drop last batch if it has only one item

### ISSUE THREAD

**Issue by remtav** [NRCan/geo-deep-learning#113]
Models with batch norm cannot handle single-item batches during training. Drop_last parameter in train Dataloader should be set to True: 
https://github.com/NRCan/geo-deep-learning/blob/master/train_model.py#L207

See: https://pytorch.org/docs/stable/_modules/torch/nn/functional.html#batch_norm

Error:
```
def batch_norm(input, running_mean, running_var, weight=None, bias=None,
               training=False, momentum=0.1, eps=1e-5):
        ...
        if size_prods == 1:
            raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))

```

**Comment by mpelchat04** [comment]
Fixed in #112
